src/index.tsx

```diff
@@ -1,14 +1,49 @@
-import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import ReactDOM from 'react-dom';
+import { createHashRouter, RouterProvider } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { NotFoundRedirect } from './pages/NotFoundRedirect';
+import { NotFound } from './pages/NotFound';
+import { Navigate } from './pages/Navigate';
+import { PeoplePage } from './pages/PeoplePage';
 
-createRoot(document.getElementById('root') as HTMLDivElement)
-  .render(
-    <Router>
-      <App />
-    </Router>,
-  );
+const router = createHashRouter([
+  {
+    path: '/home',
+    element: <Navigate />,
+  },
+  {
+    path: '/',
+    element: <App />,
+    errorElement: <NotFoundRedirect />,
```

Comment:
are you sure this is the correct way to use [errorElement](https://reactrouter.com/en/main/route/error-element#errorelement) here